src/enums/levels.ts

```diff
@@ -0,0 +1,30 @@
+import { COLOR_MAP } from 'enums/interventions';
+
+export interface LevelInfo {
+  level: string;
+  upperLimit: number;
+  name: string;
+  color: string;
+  detail: string;
+}
+
+export enum Level {
+  LOW = 'LOW',
+  MEDIUM = 'MEDIUM',
+  HIGH = 'HIGH',
+  UNKNOWN = 'UNKNOWN',
+}
+
+export const LEVEL_COLOR = {
+  [Level.LOW]: COLOR_MAP.GREEN.BASE,
+  [Level.MEDIUM]: COLOR_MAP.ORANGE.BASE,
+  [Level.HIGH]: COLOR_MAP.RED.BASE,
+  [Level.UNKNOWN]: COLOR_MAP.GRAY.BASE,
```

Comment:
Since this is defining UI colors, it would ideally live in a "content"-like directory, I think?  (trying to separate UI text / colors / thresholds / etc. into a single place)

Maybe move these into `enums/colors.ts` for now? I kinda' want to merge all the colors into a single object, like:

```
export const COLORS = {
  ALARM_LEVELS: {
    LOW: COLOR_MAP.GREEN.BASE,
    ...
  },
  CHART_SERIES: {
    PROJECTED: ...,
    ...
  },
}
```  

Then you can do `COLORS.ALARM_LEVELS.LOW` which is kinda' nice.

But that may be a future PR.